srcpkgs/lux/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'lux'
+pkgname=lux
+version=0.12.0
+revision=1
+build_style=go
+go_import_path=github.com/iawia002/lux
+depends="ffmpeg"
+short_desc="Fast, simple and clean video downloader built with Go"
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
+license="MIT"
+homepage="https://github.com/iawia002/lux"
+distfiles="https://github.com/iawia002/lux/archive/v${version}.tar.gz"
+checksum=f5bcbe1039219a299908fdd5a540052ef603ff5c8c21c0d64f44c53132c41cdd
+
+post_install() {
+	vlicense LICENSE
+}
+
+annie_package() {
+	depends="lux"
+	short_desc+=" (transitional dummy package)"
+	build_style=meta
+	}
```

Comment:
Wrong indentation.

(this is simply a nit comment, I'd appreciate it if someone else could go through the rest of the changes)

Reply:
I'm sorry, but, where is the wrong indentation? Also xlint says it's ok and "All checks have passed "